### PR TITLE
Pre release clean up

### PR DESCRIPTION
**Pull request by gmq**

### CODE REVIEW

File: src/vue-rut.spec.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+describe('Rut Filter', () => {
+  let vm;
+  beforeEach(() => {
+    Vue.filter('rut', rutFilter);
+  });
+
+  it('should format value', () => {
+    vm = new Vue({
+      template: '<div>{{superRut | rut}}</div>',
+      data: () => ({
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').

File: src/vue-rut.spec.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  });
+});
+
+describe('Rut Filter', () => {
+  let vm;
+  beforeEach(() => {
+    Vue.filter('rut', rutFilter);
+  });
+
+  it('should format value', () => {
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').

File: src/vue-rut.spec.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    const nativeInput = vm.$el.querySelector('input');
+    nativeInput.value = '7618285K';
+    nativeInput.dispatchEvent(new Event('blur'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+});
+
+describe('Rut Filter', () => {
+  let vm;
+  beforeEach(() => {
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').

File: src/vue-rut.spec.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    const nativeInput = vm.$el.querySelector('input');
+    nativeInput.value = '7618285K';
+    nativeInput.dispatchEvent(new Event('blur'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+});
+
+describe('Rut Filter', () => {
+  let vm;
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'let' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
+    nativeInput.value = '7618285K';
+    nativeInput.dispatchEvent(new Event('blur'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+});
+
+describe('Rut Filter', () => {
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').

File: src/vue-rut.spec.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    nativeInput.dispatchEvent(new Event('keyup'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+
+  it('should always format on blur even if in live mode', () => {
+    vm = new Vue({
+      template: '<div><input v-rut:live name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    nativeInput.value = '7618285K';
+    nativeInput.dispatchEvent(new Event('keyup'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+
+  it('should always format on blur even if in live mode', () => {
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').

File: src/vue-rut.spec.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+    nativeInput.dispatchEvent(new Event('keyup'));
+    expect(nativeInput.value).toBe('7618285K');
+  });
+
+  it('should format on keyup if in live mode', () => {
+    vm = new Vue({
+      template: '<div><input v-rut:live name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+      template: '<div><input v-rut name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
+    nativeInput.value = '7618285K';
+    nativeInput.dispatchEvent(new Event('keyup'));
+    expect(nativeInput.value).toBe('7618285K');
+  });
+
+  it('should format on keyup if in live mode', () => {
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').

File: src/vue-rut.spec.js [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+    nativeInput.dispatchEvent(new Event('blur'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+
+  it('shouldn\'t format on keyup by default', () => {
+    vm = new Vue({
+      template: '<div><input v-rut name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

**Comment by gmq** [comment]
[[4p/5v] Publicar vue-rut en npm](https://trello.com/c/fIBsxtTG/352-4p-5v-publicar-vue-rut-en-npm)

File: src/vue-rut.spec.js (hunk 1 above)
Comment (blackjid) [inline src/vue-rut.spec.js]: aqui el data creo que tiene sentido usar las funciones cortas en vez de arrow funcion, porque dentro de la funcion en `data` creo que uno prefiere que el `this` sea la instancia de `Vue`


Yo se que en este caso no estas usando this, pero como por acostumbrarse...

```js 
vm = new Vue({
  template: '<div>{{superRut | rut}}</div>',
  data() {
    superRut: '7618285K',
  }),
  ...
```

Comment (gmq) [inline src/vue-rut.spec.js]: Toda la razón, lo voy a actualizar.

File: src/vue-rut.spec.js [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+describe('Rut Filter', () => {
+  let vm;
+  beforeEach(() => {
+    Vue.filter('rut', rutFilter);
+  });
+
+  it('should format value', () => {
+    vm = new Vue({
+      template: '<div>{{superRut | rut}}</div>',
+      data() {
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'concise methods' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+    nativeInput.dispatchEvent(new Event('keyup'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+
+  it('should always format on blur even if in live mode', () => {
+    vm = new Vue({
+      template: '<div><input v-rut:live name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+    nativeInput.dispatchEvent(new Event('blur'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+
+  it('shouldn\'t format on keyup by default', () => {
+    vm = new Vue({
+      template: '<div><input v-rut name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
```
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

**Comment by blackjid** [comment]
aparte de que obedientemente lo cambiaste me encataria saber tu opinion... :)

File: src/vue-rut.spec.js (hunk 12 above)
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js (hunk 13 above)
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js (hunk 12 above)
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

File: src/vue-rut.spec.js (hunk 13 above)
Comment (monkeyci) [inline src/vue-rut.spec.js]: 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

**Pull request merged**